src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -119,7 +119,6 @@
     <Compile Include="System\Xml\Core\XmlSubtreeReader.cs" />
     <Compile Include="System\Xml\Core\XmlSubtreeReaderAsync.cs" />
     <Compile Include="System\Xml\Core\XmlTextEncoder.cs" />
-    <Compile Include="System\Xml\Core\BinaryCompatibility.cs" />
```

Comment:
It doesn't need a package reference for System.Threading.Tasks.Extensions. If you doesn't work with a reference then there is likely something else wrong that we might need to fix. Can you tell me what you error you get when you change it to a Reference?